eng/targetingpacks.targets

```diff
@@ -71,16 +70,11 @@
   <Target Name="RemoveFrameworkReferences"
```

Comment:
The comment above this line:

```xml
<!-- DisableImplicitAssemblyReferences support. -->
```

Seems out of date now that this code no longer references `DisableImplicitAssemblyReferences`.